browser/src/canvas/sections/HTMLObjectSection.ts

```diff
@@ -60,7 +60,7 @@ class HTMLObjectSection extends CanvasSectionObject {
 		let leftAddition = 0;
 		let topAddition = 0;
 
-		if (this.sectionProperties.objectDiv.parentNode.id === 'map') {
+		if (this.sectionProperties.objectDiv.parentNode && this.sectionProperties.objectDiv.parentNode.id === 'map') {
```

Comment:
just a design proposal to consider in the future:
can't we pass some property inside `sectionProperties` like `hasMapParent` inside concrete implementation of HTMLObjectSection which is like that? then we just verify that property here.

it's easier to track in case of changes like removing/renaming element and doesn't access DOM